configuration/analyzer_config.json

```diff
@@ -920,6 +920,39 @@
     "python_module": "tor.Tor",
     "soft_time_limit": 30
   },
+  "Triage_Scan": {
+    "type": "file",
+    "supported_filetypes":[
+      "application/x-dosexec",
+      "application/x-msdownload",
+      "application/x-msi",
+      "application/zip",
+      "application/x-7z-compressed",
+      "application/x-compressed",
+      "application/bzip2",
+      "application/x-iso9660-image",
+      "application/gzip",
+      "application/vnd.rar",
+      "application/x-tar",
+      "application/msword",
+      "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+      "application/vnd.oasis.opendocument.text",
+      "application/pdf",
+      "application/bat",
+      "application/x-javascript",
+      "application/java-archive",
+      "application/x-ms-shortcut",
+      "application/internet-shortcut"
+
+    ],
+    "description": "leverage triage sandbox environment to scan various files",
+    "python_module": "triage_scan.TriageScanFile",
+    "requires_configuration": true,
+    "additional_config_params": {
+      "api_key_name": "TRIAGE_KEY",
+      "endpoint": "public"
+    }
```

Comment:
"soft_time_limit" parameter is missing here. Default is "300s". In your experience with the APIs, how much time is required usually to perform an analysis? Consider that this could vary. Please could you try with some of the provided files in the "test_files.zip" and paste here the effective time + the report data?

Reply:
![image](https://user-images.githubusercontent.com/17424778/99374233-6364b480-28e8-11eb-8322-a0f0a8ecc78f.png)

I put the test function in both EXEs and DLLs in this case as well as testing some random files on the GUI. Time to finish varied between ~160s and ~185s

Reply:
ok so we could put the soft_time_limit explicitly to 300 anyway. Otherwise the risk is that if we change the default value we change this too, creating a possible unwanted behavior